code/FormSchemaController.php

```diff
@@ -121,7 +111,7 @@ protected function getSchemaRequested(): bool
      * @param ValidationResult $errors Required for 'error' response
      * @param array $extraData Any extra data to be merged with the schema response
      */
-    protected function getSchemaResponse(string $schemaID, ?Form $form = null, ValidationResult $errors = null, array $extraData = []): HTTPResponse
+    public function getSchemaResponse(string $schemaID, ?Form $form = null, ValidationResult $errors = null, array $extraData = []): HTTPResponse
```

Comment:
Needs to be public to avoid code duplication. See https://github.com/silverstripe/silverstripe-asset-admin/pull/1515